packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -43,8 +42,9 @@ function webpackConfigSingleSpa(opts) {
       filename: `${opts.orgName}-${opts.projectName}.js`,
       libraryTarget: "system",
       path: path.resolve(process.cwd(), "dist"),
-      jsonpFunction: `webpackJsonp_${opts.projectName}`,
+      // jsonpFunction: `webpackJsonp_${opts.projectName}`,
```

Comment:
This option was removed. Not sure if it was renamed to something else. We should look into this more.

Reply:
https://github.com/single-spa/create-single-spa/issues/198#issue-718889240

Reply:
From what I read awhile ago, it’s based on the name field in package.json

Reply:
See https://webpack.js.org/configuration/output/#outputuniquename

Reply:
Ah thanks I'll switch to that.